core/include/core/G3Constants.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _G3_CONSTANTS_H
+#define _G3_CONSTANTS_H
+
+#include <cmath>
+#include <G3Units.h>
+
+namespace G3Constants {
+	using namespace G3Units;
+
+	/* Math */
+	const double pi = M_PI;
+	const double twopi = 2.0 * M_PI;
+	const double halfpi = M_PI / 2.0;
+
```

Comment:
These are already defined constants (even the last, e.g. M_PI_2) in C. Is there a good reason to have private names for them?

Reply:
No real reason other than completeness, although the list isn't really complete anyway.  I'm fine with removing them.

Reply:
I think I would prefer to drop them if you don't mind.

Reply:
Done.